tools/m2-repo-cleaner/src/main/java/org/fusesource/ide/tools/mavenutils/M2RepoCleaner.java

```diff
@@ -120,9 +120,10 @@ private static void handleCorruptedZip(File file) {
 	}
 	
 	private static void handleLastUpdatedFile(File file) {
-		corruptedFiles.add(file.getPath());
-		if (file.isFile() && file.exists()) {
-			file.delete();
-		}
+		handleCorruptedZip(file);
+//		corruptedFiles.add(file.getPath());
```

Comment:
When does the lastUpdated file is created?
If it is at each build, deleting the whole folder will lead us to the same unstability than when cleaning the whole workspace as there will be a probability to have download corrupted jars in the .m2